### DISCUSSION TITLE
Enable Setting Sidebar Background Color From nova.php config file

### DISCUSSION THREAD

**Discussion by provydon** [laravel/nova-issues#4384] [Ideas & Feature Requests]
please could u enable setting the background Color of the Sidebar for Laravel Nova v4

we could do it back in Nova v3
<img width="1440" alt="Screen Shot 2022-06-06 at 13 54 56" src="https://user-images.githubusercontent.com/31216637/172578292-d72709ca-9c3a-4087-abfc-507c03affd1d.png">

maybe if u add "sidebar-background-color" to list of config options in nova config it would help.
<img width="408" alt="Screen Shot 2022-06-06 at 14 06 04" src="https://user-images.githubusercontent.com/31216637/172578475-5ca38552-f00f-4855-99a5-a21cdece9fb5.png">

**Comment by bashgeek** [comment]
Fairly easy doable via assets now:
https://github.com/laravel/nova-issues/discussions/4244#discussioncomment-2752877
